whisper_server/whisper_api_server.py

```
@@ -434,38 +426,38 @@ async def transcribe(request: TranscribeRequest, client_request: Request):
         audio_bytes = base64.b64decode(audio_data)
-        
+
         return {
             "audio_size_bytes": len(audio_bytes),
             "audio_size_mb": len(audio_bytes) / (1024 * 1024),
             "base64_length": len(audio_data),
             "model": request.model,
-            "language": request.language
+            "language": request.language,
         }
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)